dev-packages/e2e-tests/test-applications/nestjs/src/app.service.ts

```diff
@@ -75,6 +76,15 @@ export class AppService1 {
   async testOutgoingHttpExternalDisallowed() {
     return makeHttpRequest('http://localhost:3040/external-disallowed');
   }
+
+  @SentryTraced('wait function')
+  async wait() {
+    return new Promise(resolve => setTimeout(resolve, 500));
+  }
```

Comment:
l: We could add a second method that is not async and assert on it in the test.

Reply:
Do you mean a second non-async function and then check whether the transaction includes spans for both?

Reply:
updated